packages/babel-preset-anansi/index.js

```diff
@@ -62,7 +67,9 @@ function buildPreset(api, options = {}) {
         require('@babel/preset-react').default,
         {
           development: env !== 'production',
-          useSpread: true,
+          ...(hasJsxRuntime
+            ? { runetime: 'automatic' }
```

Comment:
ᚠ ᚡ ᚢ ᚣ ᚤ ᚥ ᚦ ᚧ ᚨ ᚩ ᚪ ᚫ ᚬ ᚭ ᚮ ᚯ ᚰ ᚱ ᚲ ᚳ ᚴ ᚵ ᚶ ᚷ ᚸ ᚹ ᚺ ᚻ ᚼ ᚽ ᚾ ᚿ ᛀ ᛁ ᛂ ᛃ ᛄ ᛅ ᛆ ᛇ ᛈ ᛉ
```suggestion
            ? { runtime: 'automatic' }
```

Reply:
I was wondering why it didn't seem to be working 🤣